library/core/src/str/converts.rs

```diff
@@ -6,6 +6,7 @@ use crate::{mem, ptr};
 
 /// Converts a slice of bytes to a string slice.
 ///
+/// This is an alias to [`std::<str>::from_utf8`].
```

Comment:
This is not a valid or correct path. `std::<str>` means module `std` with generic argument `str` (but modules can't take args and `from_utf8` is not a direct child of `std`).

The path should be:

```suggestion
/// This is an alias to [`<str>::from_utf8`].
```

Reply:
Thanks.